DOCS-#3690: Add Slack and paper link to README (#3718)
modin-project/modin: README.md

### More information and Getting Involved

- Read the [documentation](https://modin.readthedocs.io/en/latest/) for more information.

> - Check out [our paper](http://www.vldb.org/pvldb/vol13/p2033-petersohn.pdf) to learn more about how Modin works.

Reviewer: ```suggestion
- Check out [our paper](http://www.vldb.org/pvldb/vol13/p2033-petersohn.pdf) to learn more about the theory underlying Modin.
```

- Ask questions or participate in discussions on our [Discourse](https://discuss.modin.org).
- Let us know how you're using Modin! Join our community [Slack](https://modin.org/slack.html) to discuss and ask questions.

> - Join our mailing list: [modin-dev@googlegroups.com](https://groups.google.com/forum/#!forum/modin-dev).

Reviewer: ```suggestion
```
